dccrg.hpp

```diff
@@ -2618,6 +2618,13 @@ template <
 		return true;
 	}
 
+	const std::vector<std::pair<uint64_t, int>>& get_face_neighbors_of (const uint64_t cell) const
+	{
+		// Could add checks or return a nullptr / empty vector / whatever
+		// But honestly it's just better to throw
+		return face_neighbors_of.at(cell);
+	}
+
 	/*!
```

Comment:
no check here to see if for some reason the face neighbours would have become invalidated? The description of the PR sounds like there should be a call here.

Reply:
(Face) neighbor lists are updated in `update_neighbors` which is called whenever the grid is refined or load balanced, which _should_ be the only time face neighbors are invalidated. Not checking it here is consistent with `get_neighbors_of`, and both are updated in the same function.